pkg/types/ibmcloud/validation/machinepool.go

```diff
@@ -1,14 +1,23 @@
 package validation
 
 import (
+	"fmt"
+	"strings"
+
 	"github.com/IBM-Cloud/bluemix-go/crn"
 	"github.com/openshift/installer/pkg/types/ibmcloud"
 	"k8s.io/apimachinery/pkg/util/validation/field"
 )
 
 // ValidateMachinePool validates the MachinePool.
-func ValidateMachinePool(mp *ibmcloud.MachinePool, path *field.Path) field.ErrorList {
+func ValidateMachinePool(platform *ibmcloud.Platform, mp *ibmcloud.MachinePool, path *field.Path) field.ErrorList {
 	allErrs := field.ErrorList{}
+	for i, zone := range mp.Zones {
```

Comment:
This change is only tangentially related to the BZ.

Reply:
Dropped the change